attachments_component/admin/src/View/Attachments/HtmlView.php

```diff
@@ -166,9 +175,22 @@ public function display($tpl = null)
 			}
 
 		// Set the toolbar
-		$this->addToolBar();
+		if ( !$this->editor ) {
+			$this->addToolBar();
+			}
 
 		// Display the attachments
+		if ( $this->editor ) {
+			$document = Factory::getDocument();
+			HTMLHelper::_('jquery.framework');
+			$document->addScriptDeclaration('function insertAttachmentsIdToken($, editorName) {
+				let editor = parent.Joomla.editors.instances[editorName];
+				var adminform = document.getElementById("adminForm");
+				var Data = new FormData(adminform);
+				editor.replaceSelection("{attachmentsid id=" + Data.getAll("cid[]") +"}");
```

Comment:
Since we don't need to use another plugin for showing attachments by id the replaced selection should be "{attachments..." and not "{attachmentsid...".